templates/webapi/test/infopanel.html.ep

```diff
@@ -46,7 +46,10 @@
                     % else {
                         % my $state_detail = '';
                         % if (my @gru_deps = $job->gru_dependencies) {
-                            % $state_detail = join(';', map { 'id: ' . $_->gru_task_id . ', name: ' . $_->gru_task->taskname } @gru_deps);
+                            % my $task = $_->gru_task;
```

Comment:
I don't see how this is supposed to work. Did you test this?

Reply:
why not?

Reply:
Oh you are right, I wanted to put it inside the map not outside...